src/transformers/onnx/config.py

```diff
@@ -486,14 +486,18 @@ def generate_dummy_inputs(
 
         return common_inputs
 
-    def fill_with_past_key_values_(self, inputs_or_outputs: Mapping[str, Mapping[int, str]], direction: str):
+    def fill_with_past_key_values_(
+        self, inputs_or_outputs: Mapping[str, Mapping[int, str]], direction: str, inverted_values_shape: bool = False
+    ):
         """
         Fill the input_or_outputs mapping with past_key_values dynamic axes considering.
 
         Args:
             inputs_or_outputs: The mapping to fill.
             direction: either "inputs" or "outputs", it specifies whether input_or_outputs is the input mapping or the
                 output mapping, this is important for axes naming.
+            inverted_values_shape:
```

Comment:
Could we make it more clear: if `inverted_values_shape`, which axis has `past_sequence + sequence` ? 🙏

Reply:
Applied lewis' suggestion: https://github.com/huggingface/transformers/pull/19573#discussion_r994783965